.dockstore.yml

```diff
@@ -1,8 +1,8 @@
 version: 1.2
 workflows:
  - subclass: WDL
-   primaryDescriptorPath: /definitions/immuno.wdl
-   name: immuno
+   primaryDescriptorPath: /definitions/soamtic_wgs.wdl
```

Comment:
1) typo in "somatic".
2) does changing this to somatic break the immuno dockstore integration?  If so, do we need different branches or something?

Reply:
(1) ugg!
(2) I am going to rope in @tmooney  to weigh in?!

Reply:
Yeah you want to make a new entry rather than edit this one, so it ends up like:
```YAML
workflows:
  - subclass: WDL
    ...etc for immuno...
  - subclass: WDL
    ...etc for somatic_wgs...
```

Reply:
[Edited](https://github.com/wustl-oncology/analysis-wdls/pull/141/commits/e227bc79abf30f596ba7ec967101284162c4ed1c) Does this look ok?!